src/miner.cpp

```diff
@@ -433,7 +437,8 @@ CTransaction BlockAssembler::CreateSidechainWTJoinTx(uint8_t nSidechain)
 
     // Sign WT^ SCUTXO input
     const CTransaction& txToSign = mtx;
-    TransactionSignatureCreator creator(&keystoreConst, &txToSign, 0, returnAmount - amtBWT);
+    const CTransactionRef coinbaseTx = chainActive.Tip()->coinbase;
```

Comment:
This coinbase transaction should be irrelevant in this function right? There is no way a `WT^` can spend an OP_BV output right?